prisma/seed.js

```diff
@@ -0,0 +1,83 @@
+const { PrismaClient } = require("@prisma/client");
+const prisma = new PrismaClient();
+
+async function main() {
+  const alice = await prisma.user.upsert({
+    where: { email: "alice@prisma.io" },
+    update: {},
+    create: {
+      email: "alice@prisma.io",
+      userName: "AliceX1992",
+      password: "Password1234",
+      profile: {
+        create: {
+          firstName: "Alice",
+          lastName: "Hanson",
+          age: 18,
+          pictureUrl: "https://picsum.photos/200",
+        },
+      },
+    },
+    include: {
+      profile: true,
+      posts: true,
+      comments: true,
+    },
+  });
+
+
+
+  const bob = await prisma.user.upsert({
+    where: { email: "bob@prisma.io" },
+    update: {},
+    create: {
+      email: "bob@prisma.io",
+      userName: "BobIsKing",
+      password: "eatmyshorts!.1823",
+      profile: {
+        create: {
+          firstName: "Bob",
+          lastName: "Ellison",
+          age: 22,
+          pictureUrl: "https://picsum.photos/200",
+        },
+      },
+      posts: {
+        create: {
+          title: "Why JavaScript sucks",
+          content: "Because I dont understand it...",
+          imageUrl: "https://picsum.photos/200",
+          publishedAt: new Date(Date.now()),
```

Comment:
You shouldn't need to use `Date.now()`, creating a new instance of a date will use the current date/time.

On that topic, you don't even need parentheses when instantiating something without any dependencies. E.g. `publishedAt: new Date,`